crates/hotshot/src/tasks/task_state.rs

```diff
@@ -233,6 +233,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, V: Versions> CreateTaskState
             network: Arc::clone(&handle.hotshot.network),
             quorum_membership: handle.hotshot.memberships.quorum_membership.clone().into(),
             da_membership: handle.hotshot.memberships.da_membership.clone().into(),
+            drb_computations: BTreeMap::new(),
```

Comment:
do we need a `BTreeMap` here? I think we only want to run one of these calculations at a time

we can e.g. store the seeds and results in a map and trigger the start of the calculation at `block_height % epoch_height = 1` or something

Reply:
> `block_height % epoch_height = 1`

Assuming this is `(qc_block_number + 3) % task_state.epoch_height == 0` instead?

Makes sense not to use `BTreeMap`. I'm going to add a `DrbTasks` struct for this, containing the previous DRB result and the in-progress DRB calculation task, since we may need both of them in each epoch. Hopefully, it's not an overkill!

Reply:
On second thought, `BTreeMap` might be more suitable here than a `HashMap` or the new struct I proposed above. We may have at most three things to store here, two DRB results and one in-progress task. I think `BTreeMap` makes garbage collection easier because the three epochs may not be consecutive due to the membership.

Lmk if this doesn't make sense or there's a better design though!

Reply:
Fixed. The update also contains some minor changes to replace those `[0; 32]` and `[0; 96]` values with consts.

Reply:
As discussed on Zulip, I'll update the maps in a separate PR. Issue: https://github.com/EspressoSystems/HotShot/issues/3933.